templates/base/server/utils/core/logging/logger.py

```
@@ -0,0 +1,113 @@
+    "stack_info",
+    "lineno",
+    "funcName",
+    "created",
+    "msecs",
+    "relativeCreated",
+    "thread",
+    "threadName",
+    "processName",
+    "process",
+    "taskName",
+    "name",
```

Ruff (0.11.9)

Sets should not contain duplicate item `"name"`

Remove duplicate item

(B033)